### PR TITLE
OWNERS: Add an OWNERS file for the project

### PR DESCRIPTION
**Pull request by grahamwhaley**
Add the OWNERS file to give indication who reviews PRs, and who
should have final review/call on merging them.

Signed-off-by: Graham Whaley <graham.whaley@intel.com>

### CODE REVIEW

**Comment by dlespiau** [comment]
Could be a good idea to include the pullapprove file as well?

**Comment by grahamwhaley** [comment]
Ah, sure, can add pullapprove. It was not clear from the commit subject lines in the other repos that that was done as well ;-)  I'll respin and push a new version.

**Comment by grahamwhaley** [comment]
OK, pullapprove config added - but, then I guess we need to enable the project/hooks to have that process invoked as well?

**Comment by sameo** [comment]
@grahamwhaley I added the 0.7.0-clearcontainers branch to pullapprove.

**Comment by sameo** [comment]
@grahamwhaley:

```
Parsing errors

    "['clear-containers']" is an invalid value for teams: clear-containers team not found on this repo
```

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  reset_on_reopened:
+    enabled: false
+  author_approval:
+    ignored: true
+
+groups:
+  reviewers:
+    required: 1
+    teams:
+      - clear-containers
```
Comment (sameo) [inline .pullapprove.yml]: You want the `hyperstart` team here.

Comment (grahamwhaley) [reply .pullapprove.yml]: Ah, thanks. I was just trying to figure out what was upsetting it..... v3 coming soon....

Comment (sameo) [reply .pullapprove.yml]: Unless you want me to add this repo to the `clear-containers` team

Comment (grahamwhaley) [reply .pullapprove.yml]: I don't know which is best. As I took the .pullapprove file inspiration from the clearcontainers/runtime repo I'd presumed we had the same groups set up for the whole org. 
Looking at the two teams, I think the hyperstart one is fine, and obviously feels appropriate :-)

Comment (grahamwhaley) [reply .pullapprove.yml]: And now I see I probably need to change the OWNERS file to match the .pullapprove as well. If you ack that hyperstart is the correct team then I'll change the OWNERS file and re-push

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,6 @@
+reviewers:
+- clear-containers
```
Comment (sameo) [inline OWNERS]: Could you please fix that one as well ?

**Comment by grahamwhaley** [comment]
Tis done - new version.

**Comment by sameo** [comment]
Approved

**Pull request merged**